gp-templates/glossary-import.php

```diff
@@ -11,7 +11,7 @@
 
 <h2><?php _e( 'Import Glossary Entries', 'glotpress' ); ?></h2>
 <p>
-	<?php printf( __( 'Use this form to bulk upload glossary entries. The entries should be stored in a CSV file, matching the custom glossary format from <a href="%s">Google Translator Toolkit</a>.', 'glotpress' ), 'https://support.google.com/translate/toolkit/answer/147854' ); ?><br/>
+	<?php printf( __( 'Use this form to bulk upload glossary entries. The entries should be stored in a CSV file with a custom glossary format. Read more on <a href="%s">Polyglots Handbook</a>.', 'glotpress' ), 'https://glotpress.blog/the-manual/glossaries/' ); ?><br/>
```

Comment:
"Polyglots Handbook" should have been replaced with "GlotPress handbook"

Reply:
@ocean90 good point. This PR was closed - I just checked "Allow edits from maintainers." so that'll make it easier to update the text?
Sorry to take your time with this small change... 🙏

Reply:
I've created a new PR for this and will merge it.